docs/en/ingest-management/elastic-agent/configuration/env/shared-env.asciidoc

```diff
@@ -560,4 +560,20 @@ contains your CA's certificate.
 
 // end::kibana-ca[]
 
+// tag::netinfo[]
+|
+[id="env-{type}-netinfo"]
+`NETINFO`
```

Comment:
Seems this changed in the PR to ELASTIC_NETINFO?

Reply:
Updated. Was in the middle of the changes 👍